src/Components/Credits/LoanAmortizationSimulation/FunctionForTest/test/loanAmortizationResult.test.ts

```diff
@@ -0,0 +1,50 @@
+import { Dispatch } from '@reduxjs/toolkit'
+import loanAmortizationResult from '../loanAmortizationResult.ts'
+import {
+  // updateResult,
+  resetResult,
+} from '../../loanSimulationAmortizationResultState.ts'
+
+const dispatch: Dispatch = jest.fn()
+
+// loanValue: number
+// commisionFee: number
+// interestForBasePeriod: number
+// totalPaymentPeriods: number
+// doesTheBankChargeACommission: string
+// interestAccrualMethod: string
+// initialDebtBalanceArr: number
+// dispatch: Dispatch
+
+beforeEach(() => {
+  jest.clearAllMocks()
+})
+
+// TODO: tu mam taki problem że loanValue w teorii może być 0, NaN lub ujemny ale wtedy zawsze initialDebtBalanceArr powinna mieć długość zero z podkreśleniem na powinna, pytanie czy moge to jakoś dynamicznie zaimportować taką logike czy może musze sie opierać na testach innych funckji albo zmienić zabezpieczenie przed takim incydentem w samej funkcji loanAmortizationResult? (problem dotyczy oczywiście wszyskich innych wartości) Masz może jakieś dobre kursy o testowaniu do polecenia?
+
```

Comment:
Unfortunately I don't have  courses (I'm not good at testing to be honest). You should include in your function checking if `initialDebtBalanceArr` is `0` when `loanValue ` is incorrect